.github/workflows/build-citus-community-nightlies.yml

```diff
@@ -27,7 +27,8 @@ jobs:
         platform:
           - el/7
           - el/8
-          - ol/7
+          # temporarily removed from nightlies since pg15 beta is not being downloaded for ol/7
+          # - ol/7
```

Comment:
Do we not create packages for all PG versions? If only PG15 does not work for ol/7 is there a way to disable only this combination?

Reply:
There is no way for it now. I opened the PR in tools to be able to handle it
https://github.com/citusdata/tools/pull/238

Reply:
This PR is to enable nightlies without waiting the merge od this enablement PR. After merge of this PR there will be no need to disable ol/7 . We will just exclude pg15